.circleci/publish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+CURRENT_TAG=$(git tag -l --points-at HEAD)
+
+if [[ ! "$CURRENT_TAG" =~ v* ]]; then
+    echo "publish can be run only on v* tag"
+    exit 1
+fi
+
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [[ "$CURRENT_BRANCH" != "master" ]]; then
```

Comment:
I'm not sure if `rev-parse` shows `master`
According to 
```
       filters:
          tags:
            only: /^v*/
          branches:
            ignore: /.*/
```
Circle CI will checkout tag, not branch. 
In this case on my machine list of branches containing checkouted commit are displayed only with `--branches` arg

Reply:
thanks, i'll fix that